packages/api/main.go

```diff
@@ -136,6 +135,33 @@ func NewGinServer(ctx context.Context, logger *zap.Logger, apiStore *handlers.AP
 			}),
 	)
 
+	r.Use(
+		// Request logging must be executed after authorization (if required) is done,
+		// so that we can log team ID.
+		customMiddleware.ExcludeRoutes(
+			func(c *gin.Context) {
```

Comment:
we need to call gin zap logger manually because in we need to extract team in-request